packages/components/src/Chip/Chip.types.d.ts

```diff
@@ -0,0 +1,34 @@
+import React, { PropsWithChildren } from "react";
+
+export interface ChipProps extends PropsWithChildren {
+  /** Accessible Label */
+  ariaLabel?: string;
+  /** Disables Chip */
+  disabled?: boolean;
+  /** Adds a Medium Heading with a Pipe separator */
+  heading?: string;
+  /** Makes Chip red. */
```

Comment:
I would be a smidge more semantic in these descriptions so if (when) the styling related to these properties change we don't need to do too much adjusting here. For example, "makes Chip red" could maybe be "Changes Chip styling to inform the user of an error"? This maybe isn't the _best_ example because red is so "error-y" but maybe one day we decide to add "help text" to an invalid Chip that would still leverage this prop... does that make sense?

Reply:
Heading is another example, we probably don't need to get specific to the font weight, so much as "adds more prominent text to act as a heading"

Reply:
Totally makes sense in both cases. I did this pretty late last night and I think I was trying to be a bit clever in both cases but it came off lazy instead. 

Also in the habit of having anything written that will be seen by others be double checked by a copywriter, but I need to break myself of that habit. :)